templates/method/drupal-cms.hbr

```diff
@@ -138,10 +154,18 @@
                             {{/required}}
                         </td>
                         <td>
-                            {{#if required}}
-                            <input type="text" class="form-control" data-role="value" value="{{defaultValue}}"/>
+                            {{#if options}}
+                                <select name="{{name}}" data-role="multiple-value">
+                                    {{#options}}
+                                        <option value="{{.}}">{{.}}</option>
```

Comment:
Two comments:
1. If there is a default value, and it is not the first item in the list, we will need to set it as default via jQuery. I'd say put a data-defaultValue attribute on the select. This will involve augmentation of model.js.
2. If the field has options but is not required, the first option in the dropdown should be "(no value)" or something similar.